omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ModuleController1_8Test.java

```diff
@@ -102,7 +102,46 @@ public void shouldIncludeVersionToDefaultRepresentation() throws Exception {
 		
 		Assert.assertNotNull(PropertyUtils.getProperty(result, "version"));
 	}
+
+	@Test
+	public void shouldGetAFullRepresentation() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		req.addParameter("v", "full");
+		SimpleObject result = deserialize(handle(req));
+		
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "uuid"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "display"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "name"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "description"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "packageName"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "author"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "version"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "started"));
+	}
+	
+	@Test
+	public void shouldGetADefaultRepresentation() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
```

Comment:
The name does not still reflect what is being tested. For instance, if the default representation was returned without a uuid or any other property, would this test name reflect that?